src/messaging/mod.rs

```diff
@@ -640,7 +640,7 @@ mod tests {
     }
 
     #[test]
-    fn try_from() -> Result<()> {
+    fn try_from() -> anyhow::Result<()> {
```

Comment:
Perhaps simply `use anyhow::{Result, anyhow}` for the whole `test` module?

Reply:
I did try that and things got wiiild w/ compiler errors. I'll double check that's not sorted now though

Reply:
that wildness is only in the seq file. the rest ill update